task_4_exceptions
